AFNetworking/AFHTTPClientSKZ.m

```diff
@@ -674,6 +674,19 @@ - (void)enqueueBatchOfHTTPRequestOperations:(NSArray *)operations
     [self.operationQueue addOperation:batchedOperation];
 }
 
++ (dispatch_queue_t)backgroundQueue
+{
+    static dispatch_once_t pred;
+    static dispatch_queue_t backgroundQ;
+    
+    dispatch_once(&pred, ^{
+        backgroundQ = dispatch_queue_create("background_http_queue", DISPATCH_QUEUE_SERIAL);
```

Comment:
Are you sure we want this to be serial? You're removing the current serial bottleneck by switching the CDM to block based away from the FIFO queue it's been on, right? Also, we currently have the SKZSkillzAPIClient setup to handle 10 concurrent operations. Seems like it adds a new bottleneck?